src/main/java/se/sundsvall/alkt/api/model/Owner.java

```diff
@@ -0,0 +1,123 @@
+package se.sundsvall.alkt.api.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder(setterPrefix = "with")
+@AllArgsConstructor
+@NoArgsConstructor
+public class Owner {
+
+	@Schema(description = "Organization number", example = "1234567890")
+	private String organizationNumber;
+
+	@Schema(description = "Name of the organization", example = "Sundsvalls kommun")
+	private String organizationName;
+
+	@ArraySchema(schema = @Schema(description = "List of all establishments / objects", implementation = AlktObject.class))
+	private List<AlktObject> objects;
+
+	@Getter
+	@Setter
+	@Builder(setterPrefix = "with")
+	public static class AlktObject {
+
+		@Schema(description = "Name of the establishment / object", example = "Sundsvalls kommuns bar")
+		private String name;
+
+		@Schema(description = "When the establisment /object was first posted", example = "2021-01-01T00:00:00")
+		private LocalDateTime postedDate;
+
+		@Schema(description = "When the establishment / object changed", example = "2021-01-01T00:00:00")
+		private LocalDateTime changedDate;
```

Comment:
changed? 

(You don't have the data type in the other variable names either, I.e. there is no "nameString" variable, just a "name" variable)

Reply:
Hm, must be "autocorrect".